include/aos/common/crypto/mbedtls/cryptoprovider.hpp

```diff
@@ -165,6 +165,15 @@ class MbedTLSCryptoProvider : public x509::ProviderItf {
      */
     Error ASN1DecodeOID(const Array<uint8_t>& inOID, Array<uint8_t>& dst) override;
 
+    /**
+     * Creates SHA1 hash based on a given name space and data.
+     *
+     * @param namespace name space.
```

Comment:
there's no 'namespace' param. Please make it consistent.

Reply:
done